packages/valory/connections/ledger/ledger_dispatcher.py

```diff
@@ -264,7 +264,7 @@ def get_transaction_receipt(
                     message.transaction_digest.body,
                     raise_on_try=True,
                 )
-            except Exception as e:  # pylint: disable=broad-except  # pragma: nocover
+            except Exception as e:  # pylint: disable=broad-except
```

Comment:
Try to avoid `# pragma: nocover`, it makes our tests weaker.

Reply:
created a [Trello ticket](https://trello.com/c/8G17Vn54/894-remove-pragma-nocover-from-repositories) to remove such occurrences from our code base

Reply:
@Adamantios I know you checked for other cases in this repository that might be affected and found none. Did you also investigate the [open-aea ledger dispatcher](https://github.com/valory-xyz/open-aea/blob/main/packages/fetchai/connections/ledger/ledger_dispatcher.py)?

Reply:
No, I did not. I guess [this](https://github.com/valory-xyz/open-aea/blob/main/packages/fetchai/connections/ledger/ledger_dispatcher.py#L235-L262) has the same issue right?

Reply:
Opened https://github.com/valory-xyz/open-aea/issues/232.